hw/ip/sysrst_ctrl/doc/theory_of_operation.md

```diff
@@ -159,9 +159,9 @@ Note that since the `sysrst_ctrl` does not have control over the pad open-drain
 ## Flash Write Protect Output
 
 Upon reset, the `flash_wp_l_o` signal will be asserted active low.
-The software can release `flash_wp_l_o` explicitly by setting [`PIN_OUT_CTL.FLASH_WP_L`](registers.md#pin_out_ctl) to 0 when needed.
 The `flash_wp_l_o` signal does have a corresponding input signal `flash_wp_l_i` - but that one is mainly intended for pad observability and does not have a bypass path to `flash_wp_l_o`.
 Hence, the value of `flash_wp_l_o` defaults to logic 0 when it is not explicitly driven via the override function.
```

Comment:
```suggestion
Hence, the value of `flash_wp_l_o` defaults to 0V when it is not explicitly driven via the override function.
```

I think the phrase `logic 0` for a active low signal is a little ambious. Can we use `0V` or something else like that?

Reply:
No, physical values would not be correct, since the voltage is an implementation-dependent signaling property. That does not belong in block-level documentation. I would probably go with "asserted low".

Really, though, the CSR documentation is unclear because it describes flash_wp_l controls as overrides, and that's nonsensical. It's just a GPIO that is assumed to be connected to an open-drain pad.

Reply:
I think "asserted low" is a great phrase to use that is clear.

Reply:
I have replaced `logic 0` with `asserted low`.

Reply:
For reference of the corresponding RTL, the "non-override" value is the constant here:

https://github.com/lowRISC/opentitan/blob/908eefb9a5888c55c9baac8a6a2780746c3afe4d/hw/ip/sysrst_ctrl/rtl/sysrst_ctrl_pin.sv#L99